### PR TITLE
Performance support update

### PR DESCRIPTION
**Pull request by techninja**
This adds support for the latest CNCServer 2.5.0 update, among many other performance related updates.

### CODE REVIEW

File: resources/_i18n/robopaint.en-US.json [hunk 1]
```diff
@@ -244,8 +255,8 @@
     "output": {
       "skipwhite": "Skip White Paint",
       "skipwhiteinfo": "When your media settings (paint colors) contain white, ignore any paths that would be painted in that color. You may want to turn this off when painting on a background that is not white.",
-      "prefill": "Prefill buffer",
-      "prefillinfo": "If enabled, all batch commands on supporting modes will be sent before the robot begins work. This should eliminate stuttering on start, though you have to wait longer before your work begins printing.",
+      "nativerunner": "Use Native CNCServer Runner",
+      "nativerunnerinfo": "If enabled, CNCServer will stream commands via a separate node.js process that will be immune to Electron's background limitations. **Change requires App restart, and node.js installed globally.**",
```
Comment (docprofsky) [inline resources/_i18n/robopaint.en-US.json]: Do we still need the native runner since #253 fixes this problem?

Comment (techninja) [inline resources/_i18n/robopaint.en-US.json]: Probably not, though it seems like a nice option for testing.. or whatever.

**Comment by forresto** [comment]
Can we get a beta release?

**Comment by oskay** [comment]
@forresto We probably will not have an official update to the beta just yet, but may have an "unofficial compile" available for you to try soon.

**Pull request merged**